conda_forge_tick/migrators/arch.py

```diff
@@ -413,3 +413,150 @@ def pr_body(self, feedstock_ctx: ClonedFeedstockContext) -> str:
 
     def remote_branch(self, feedstock_ctx: FeedstockContext) -> str:
         return super().remote_branch(feedstock_ctx) + "_arm_osx"
+
+
+class WinArm64(GraphMigrator):
+    """
+    A Migrator that add win-arm64 builds to feedstocks
+    """
+
+    migrator_version = 1
+    rerender = True
+    # We purposefully don't want to bump build number for this migrator
+    bump_number = 0
+    ignored_packages = {}
+    arches = {"win_arm64": "win_64"}
+
+    def __init__(
+        self,
+        graph: nx.DiGraph = None,
+        name: Optional[str] = None,
+        pr_limit: int = 0,
+        piggy_back_migrations: Optional[Sequence[MiniMigrator]] = None,
+        target_packages: Optional[Sequence[str]] = None,
+        effective_graph: nx.DiGraph = None,
+        _do_init: bool = True,
+    ):
+        if _do_init:
+            if target_packages is None:
+                # We are constraining the scope of this migrator
+                with open(
+                    os.path.join(
+                        os.environ["CONDA_PREFIX"],
+                        "share",
+                        "conda-forge",
+                        "migrations",
+                        "win_arm64.txt",
```

Comment:
We need to add this file to the pinnings repo, even if it is empty for now, before we can merge this PR.

Reply:
I believe that already exists! https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/win_arm64.txt